plugins/home/README.md

```diff
@@ -0,0 +1,106 @@
+
+Finally, the `createCardExtension` also accepts a generic, such that Component Developers can indicate to App Integrators what custom props their component will accept, such as the example below where the default category of the random jokes can be set.
+
+```tsx
+export const RandomJokeHomePageComponent = homePlugin.provide(
+  createCardExtension<{ defaultCategory?: 'programming' | 'any' }>({
+    title: 'Random Joke',
+    content: () =>
+      import('./homePageComponents/RandomJoke/Content').then(m => m.Content),
+    actions: () =>
+      import('./homePageComponents/RandomJoke/Actions').then(m => m.Actions),
+    contextProvider: () =>
+      import('./homePageComponents/RandomJoke/Context').then(
+        m => m.ContextProvider,
+      ),
+    settings: () =>
+      import('./homePageComponents/RandomJoke/Settings').then(m => m.Settings),
+  }),
+);
+```
+
+In summary: it is not necessary to use the `createCardExtension` extension creator to register a home page component, although it is convenient since it provides error boundary and lazy loading, and it also may hook into other functionality in the future.
+
+### Composing a Home Page
+
+Composing a Home Page is no different from creating a regular React Component, i.e. the App Integrator is free to include whatever content they like. However, there are components developed with the Home Page in mind, as described in the previous section. If created by the `createCardExtension` extension creator, they are rendered like so
+
+```tsx
+import React from 'react'
+import Grid from '@material-ui/core/Grid'
+import { RandomJokeHomePageComponent } from '@backstage/plugin-home';
+
+export const HomePage = () => {
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12} md={4}>
+        <RandomJokeHomePageComponent>
+      </Grid>
+    </Grid>
+  )
+}
+```
+
+Additionally, the App Integrator is provided an escape hatch in case the way the card is rendered does not fit their requirements. They may optionally pass the `Renderer`-prop, which will receive the `title`, `content` and optionally `actions`, `settings` and `contextProvider`, if they exist for the component. This allows the App Integrator to render the content in any way they want.
+
+## Contributing
+
+We believe that people have great ideas for what makes a useful Home Page, and we want to make it easy for every to benefit from the effort you put in to create something cool for the Home Page. Therefore, a great way of contributing is by simply creating more Home Page Components, than can then be used by everyone when composing their own Home Page.
```

Comment:
> simply creating more Home Page Components

worth to mention anything about where these home page components should live?

Reply:
Yes, good call! :rocket: